### PR TITLE
Simplify extensions class names concatenation + Add a test for extensions combination

### PR DESCRIPTION
**Pull request by bitxon**

### CODE REVIEW

File: src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerExtensionTest.java [hunk 1]
```diff
@@ -43,7 +43,11 @@ public class WireMockContainerExtensionTest {
     public WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
             .withStartupTimeout(Duration.ofSeconds(60))
             .withMapping("json-body-transformer", WireMockContainerExtensionTest.class, "json-body-transformer.json")
-            .withExtension("JSON Body Transformer", Collections.singleton("com.ninecookies.wiremock.extensions.JsonBodyTransformer"),
+            .withExtension("Webhook",
```
Comment (oleg-nenashev) [inline src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerExtensionTest.java:46]: Maybe a separate test then?

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Thanks for the patch. While it generally makes the code a bit easier, iIMHO it does not justify pulling in a dependency on WireMock, and creating conflict risks for downstream dependencies. I would rather look for a similar functionality in TestContainers API, or include a shared library like Apache Commons with https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html

File: src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerExtensionTest.java (hunk 1 above)
Comment (bitxon) [reply src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerExtensionTest.java]: Make sense. Code with two extensions moved to separate test class

**Review by oleg-nenashev — APPROVED** [review]
As I mentioned on Slack, it was my mistake during the review. `wiremock-webhooks-extension` is downloaded in `dependency:get` and hence there is no impact on classpath at alll. Thanks @bitxon!

**Pull request merged**